pkgs/tools/filesystems/sshfs-fuse/default.nix

```
@@ -1,52 +1,30 @@
-{ lib, stdenv, fetchFromGitHub
-, meson, pkg-config, ninja, docutils, makeWrapper
-, fuse3, glib
-, which, python3Packages
-, openssh
-}:
+{ lib, stdenv, callPackage, fetchpatch, darwin }:
```

```suggestion
{ lib, stdenv, callPackage, fetchpatch, DiskArbitration }:
```

and add ``inherit (darwin.apple_sdk.frameworks) DiskArbitration;`` to the top level entry.